README.md

```diff
@@ -7,10 +7,18 @@ Please fork this repository to your own account, either directly edit the README
 Your statements should include:
 
 * The problem you are addressing
-* The eductional goal that you plan to address
-* Your objectives for the project
+
+How can we use existing student data in combination with student interaction data to assess, measure, and predict performance on state mandated assessment examinations?
+
+* The educational goal that you plan to address & objectives for the project
+ 
+I would like to use machine learned methods on existing student interaction data to first extract knowledge component data, and then use these KC values in order to predict their performance on a future knowledge skill assessment. Predictive modeling will be accomplished by aggregating this KC level data with student demographics and past achievement on previous exams. 
+
 * Your priorities for the project
+
+Obtain a dataset, analyze and clean the dataset, generate the KC features, aggregate KC features with student data, build model using ML, score and evaluate model, back test model performance and evaluate student success performance. 
```

Comment:
Looks doable. Where do you plan to get the data? Have you checked out LearnShpere yet?